circle.yml

```diff
@@ -219,7 +224,7 @@ workflows:
       - build-cxx17
       - build-cxx14-asan
       - build-gcc6
-      - build-clang3.8
+      - build-clang38
```

Comment:
Why change the name of this?

Reply:
In CircleCI 2.1 you cannot have dots in the job names.